documentation/src/main/markdown/integrations/jenkinsplugin/pluginreleasenotes.md

```diff
@@ -1,5 +1,22 @@
 # Release Notes for Jenkins Plugin
 
+## **Version 9.0.0**
+**New features**
+
+* Updated to be compatible with [solution_name] 9.x.x. (Downloading and using detect9.(sh/ps1)).
+
+**Changed features**
+
+* The Jenkins plugin has been upgraded to use [solution_name] 9.x.x for execution.
```

Comment:
* Updated to use [solution_name] 9.x.x for execution.
* Jenkins version 2.401.3 or later is required.

Change this line to a note:
* Configuration and usage of the plugin is unchanged.
re:
<note type="note">Configuration and usage of the plugin is unchanged.</note>